client/driver/lxc.go

```diff
@@ -168,6 +168,10 @@ func (d *LxcDriver) Fingerprint(cfg *config.Config, node *structs.Node) (bool, e
 	return true, nil
 }
 
+func (d *LxcDriver) Prestart(ctx *ExecContext, task *structs.Task) error {
+	return nil
```

Comment:
Shouldn't we be setting the env vars here since the paths will be rooted in the container?